### PR TITLE
paths to block notation

### PR DESCRIPTION
**Pull request by southeo**
- field and class selectors
- batch metadata

Also updated [documentation for mas devs](https://github.com/DiSSCo/dissco-developers-documentation/wiki/Information-for-Machine-Annotation-Service-(MAS)-Developers) 
[
jira](https://naturalis.atlassian.net/browse/DD-1363?atlOrigin=eyJpIjoiY2I5Njc5YjU2YWI4NDUzY2JjMjU0ZDRhYjBjZjExMzEiLCJwIjoiaiJ9)

### CODE REVIEW

File: osm-georeferencing/main.py [hunk 1]
```diff
@@ -72,11 +71,11 @@ def map_to_annotation_event(specimen_data: Dict, results: List[Dict[str, str]],
...
                 results))
     annotation_event = {
-        "jobId": job_id,
-        "annotations": annotations
+        'jobId': job_id,
+        'annotations': annotations
     }
     if batching:
-        annotation_event["batchMetadata"] = batch_metadata
+        annotation_event['batchMetadata'] = batch_metadata
```
Comment (TomDijkema) [inline osm-georeferencing/main.py:78]: I think you can just use dot notation here :)

File: osm-georeferencing/main.py [hunk 2]
```diff
@@ -11,11 +10,11 @@
...
 
 logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
-OA_BODY = "oa:hasBody"
-DWC_LOCALITY = "dwc:locality"
-OA_VALUE = "oa:value"
-LOCATION_PATH = "$.ods:hasEvent[*].ods:Location."
-USER_AGENT = "Distributed System of Scientific Collections"
+OA_BODY = 'oa:hasBody'
+DWC_LOCALITY = 'dwc:locality'
+OA_VALUE = 'oa:value'
```
Comment (TomDijkema) [inline osm-georeferencing/main.py:15]: Nice! How did this not fail earlier? Because it were strings without quotes

**Review by TomDijkema — COMMENTED** [review]
Nicely done!

File: osm-georeferencing/main.py (hunk 2 above)
Comment (southeo) [reply osm-georeferencing/main.py:15]: no it was double quotes i changed to single quotes to be more consistent

File: osm-georeferencing/main.py (hunk 1 above)
Comment (southeo) [reply osm-georeferencing/main.py:78]: sadly no because this is also adding the `batchMetadata` key to the array. No dots here

Comment (TomDijkema) [reply osm-georeferencing/main.py:78]: But that works exactly the same with dots if I am not mistaken

**Review by samleeflang — APPROVED** [review]
:juggling_person:

**Pull request merged**